allure-web/.gitignore

```diff
@@ -3,3 +3,4 @@ coverage
 etc
 node_modules
 node
+yarn.lock
```

Comment:
I have already tried that in other projects. It causes merge conflicts for no reason. This problem is bigger than benefits from fixed package versions